frontend/pages/add_salary.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div>
+    <main class="site__main pt-5 lg:pt-0 mb-10 md:mb-20">
+      <div class="container mx-auto w-10/12">
+        <Title title="100% Anonymous" fontSize="header" onlyTitle="onlyTitle" />
```

Comment:
hi Asam 
did you test and make sure it is working??
i'm wandering because you are using the Title Component in this Component widthout importing it.

Reply:
Yes, it actually does. Indeed, I activated the auto import option of Nuxt JS (in the nuxt.config.js file) and therefore I do not need to specify the imports 

![autoimport](https://user-images.githubusercontent.com/34966088/151294577-6fff3a06-6eb0-46db-9526-1b5d459b6ab3.png)